home.html

```diff
@@ -0,0 +1,130 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Megaphone - Announce your life home page">
+    <link rel="stylesheet" href="dist/css/styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+</head>
+<body>
+    <nav class="navbar navbar-expand-md bg-body-tertiary">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="home.html">Megaphone <i class="bi bi-megaphone-fill"></i></a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="./profile.html">Profile</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                  More
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Account</a></li>
+                  <li><a class="dropdown-item" href="#">Global</a></li>
+                  <li><hr class="dropdown-divider"></li>
+                  <li><a class="dropdown-item" href="./index.html">Log out</a></li>
+                </ul>
+              </li>
+            </ul>
+            <form class="d-flex" role="search">
+              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+              <button class="btn btn-outline-success" type="submit">Search</button>
+            </form>
+          </div>
+        </div>
+      </nav>
+        <section class="container position-relative mb-4 mt-4">
+            <div class="row">
+              <h2>Create post</h2>
+              <form class="col">
+                <div class="mb-3">
+                  <label for="exampleFormControlTextarea1" class="form-label">Write something..</label>
+                  <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+                </div>
+                <button type="submit" class="btn btn-primary">Submit</button>
+                <i class="bi bi-paperclip"></i><span>Add file</span>
+              </form>
+            </div>
+          </section>
+          <section class="container">
+            <div class="row">
+              <h2>Wall posts</h2>
+                <div class="card p-4 mb-3 col-12 col-md-6 col-lg-4">

```

Comment:
Please use a code formatter to indent your code automatically.

Reply:
I've had this installed for a while, but I just found my formatter settings where set to null.. Thanks for making me investigate this further. Actually, when searching for format in settings, the default formatter doesn't show.. removed the last letter in search term, and there it is...